src/site/content/en/learn/css/box-model/index.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: course
+title: Box Model
+authors:
+  - andybell
+tags:
+  - css
```

Comment:
Items with tags will show up in our /tags/ pages. @una @argyleink is that something y'all want? I could see an argument either way (it helps with course discovery but it also drops someone right in the middle of things).